packages/gasket-plugin-intl/lib/index.d.ts

```diff
@@ -55,6 +56,8 @@ declare module '@gasket/data' {
   export interface GasketData {
     intl?: {
       locale: string;
+      /* dynamic import attributes toggle */
+      experimentalImportAttributes?: boolean;
```

Comment:
You're not adding this via the publicGasketData hook, and I don't see it as necessary anyhoo
```suggestion
```